### PR TITLE
Create CRUD of movement remappings

### PR DESCRIPTION
**Pull request by dmitryrck**

### CODE REVIEW

**Comment by dmitryrck** [comment]
I am not too happy with the name, suggestions are welcome.

File: app/models/movement_remapping.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class MovementRemapping < ApplicationRecord
```
Comment (rafaelrpbelo) [inline app/models/movement_remapping.rb:1]: `MovementReplacement` ? :thinking:

File: spec/features/admin/movement_remappings_spec.rb [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      click_on "Edit"
+
+      fill_in "Order", with: "300"
+
+      click_on "Update"
+
+      expect(page).to have_content "ORDER 300"
+    end
+
+    it "comments" do
```
Comment (rafaelrpbelo) [inline spec/features/admin/movement_remappings_spec.rb:67]: I can see that is being repeated around the features' specs.
Maybe we can create shared examples or even a helper.

**Review by rafaelrpbelo — APPROVED** [review]
It seems great to me! :tada:

**Pull request merged**